templates/favourite/css/custom.css

```diff
@@ -808,6 +808,28 @@ video.responsive-vid {
 }
 /* ### END TWITTER WIDGET ### */
 
+/** ### CSS for the citation project ### **/ 
+.radio-button input[type="radio"],
+input.radio {
+  float: left;
+  clear: none;
+  margin: 2px 0 0 2px; 
+}
+
+.radio-button label {
+  float: left;
+  clear: none;
+  display: block;
+  padding: 0px 1em 0px 8px;
+  margin: 2px 0 0 0; /** this is causing issues. This postions the labels correctly locally but not on dev. on dev, not setting this attribute fixes the issue..**/
```

Comment:
I'll be needing your help on this one @gsviteri It seems that the buttons only appear misaligned when I view dev on Chrome. They appear okay on Frirefox. Also, the order in which the style sheets are being applied is different between dev and local